cmd/node/factory/structsStatusHandlers.go

```diff
@@ -121,6 +124,67 @@ func (shi *statusHandlersInfo) UpdateStorerAndMetricsForPersistentHandler(store
 	return nil
 }
 
+// LoadTpsBenchmarkFromStorage will try to load tps benchmark from storage or zero values otherwise
+func (shi *statusHandlersInfo) LoadTpsBenchmarkFromStorage(store storage.Storer, marshalizer marshal.Marshalizer) *statistics.TpsPersistentData {
+	emptyTpsBenchmarks := &statistics.TpsPersistentData{
+		BlockNumber:           0,
+		RoundNumber:           0,
+		PeakTPS:               0,
+		AverageBlockTxCount:   big.NewInt(0),
+		TotalProcessedTxCount: big.NewInt(0),
+		LastBlockTxCount:      0,
+	}
+	lastNonceBytes, err := store.Get([]byte(core.LastNonceKeyMetricsStorage))
+	if err != nil {
+		log.Trace("cannot load last nonce from metrics storage", "error", err, "key", []byte("lastNonce"))
```

Comment:
TPS benchmarks are not very important and also they are only relevant for metachain nodes. However changed to debug as it can show us possible issues with the storage